test/openapi2/request/yaml/pet-store-swagger-formats.yaml

```diff
@@ -0,0 +1,94 @@
+swagger: "2.0"
```

Comment:
Any idea why there are so many yaml files?

Reply:
Because there are specific things you want to check and by this why it more clearer